upgrading/topics/migrate_db.adoc

```diff
@@ -32,6 +32,33 @@ Or run this CLI command:
 When you start the server with this setting your database is automatically migrated if the database schema has changed
 in the new version.
 
+Creating an index on huge tables with millions of records can easily take a huge amount of time
+and potentially change major service disruption on upgrades.
+For those cases, we added the threshold (the number of records) for automated index creation.
+When the number of records is higher than the threshold, the index is not created automatically,
+and there will be a warning message in server logs including SQL commands which can be applied later manually.
+
+To change the threshold, set the indexCreationThreshold property, value for the default connectionsLiquibase provider:
+
+[source,xml]
+----
+<spi name="connectionsLiquibase">
+    <provider name="default" enabled="true">
+        <properties>
+            <property name="indexCreationThreshold" value="10000"/>
```

Comment:
```suggestion
            <property name="indexCreationThreshold" value="10000"/>
```

Reply:
Seems I made a typo myself in the suggestion but you got it right and fixed to `100000`